src/views/doc/OpenAPIOverview/index.tsx

```diff
@@ -63,7 +63,7 @@ interface IComponentState {
     isConnectionEditDialogOpen: boolean;
     isComponentEditDialogOpen: boolean;
     connectionToEdit?: IConnectionEntity | undefined;
-    componentToEdit?: IComponentEntity | undefined;
+    componentToEdit?: IComponent | undefined;
```

Comment:
isn't it needed to have a dedicated IComponent and IComponentOpenApi ?

Reply:
Yes I planned to refactor the types, to have dedicated type for component and component open api, must I add this task to the backlog ?

Reply:
yes, please add it to the backlog